src/Controller/Admin/BaseDTOControllerTrait.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Compotes package.
+ *
+ * (c) Alex "Pierstoval" Rock <pierstoval@gmail.com>.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Controller\Admin;
+
+use App\Form\DTO\EasyAdminDTOInterface;
+use App\Form\DTO\TranslatableDTOInterface;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\EasyAdminController;
+use Gedmo\Translatable\Entity\Repository\TranslationRepository;
+use InvalidArgumentException;
+use RuntimeException;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+
+trait BaseDTOControllerTrait
```

Comment:
I intend to work on DTOs support again on EA3.

Let's see if it has enough power for it 🙂